docs/apidocs/index.rst

```diff
@@ -0,0 +1,13 @@
+******************************
+qiskit-addon-obp API reference
```

Comment:
Leaving the package name like this seems odd to me. I would do any one of the following:

- emphasize that this is not a word but a code keyword: `qiskit-addon-obp`
- change it from a lowercase-hyphenated word to be the package description: Qiskit Addon OBP

Reply:
Sure. I'm following the convention Qiskit, Runtime, and Transpiler Service already do of using the package name. I think the reason is that the left sidebar will already call it "Operator backpropagation".

I'll use code syntax.